build.gradle

```diff
@@ -266,7 +266,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.1'
+        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.1_RDM-13132_3'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see hmcts/ccd-test-definitions#43.